aws/modules/deploy/charts/config.yml

```diff
@@ -7,11 +7,12 @@
 # Child directory named "values" may contain ERB template with custom values
 # for the chart, that is being rendered in `rake generate` task.
 #
-# Every chart should have the following attributes:
+# Every chart may have the following attributes:
 #
 # beautiful-chart:
-#  release_name: release-name-to-pass-to-helm
-#  release_namespace: release-namespace-to-pass-to-helm
+#  release_name: release-name-to-pass-to-helm              – optional, defaults to chart name
+#  release_namespace: release-namespace-to-pass-to-helm    - optional, defaults to "default" namespace
+#  allow_upgrade: false                                    - optional, default is false
```

Comment:
Sorry, @natarajaya but the logic confuses me a bit:

if default is false, it means that by default all the charts are not allowed to be upgraded, unless you set it to true but then in deploy.rake, if the value is true is when the helm is updated. Wouldn't it be clearer if the default value is `true`?

Reply:
Yep, mistake in the comment – default is `true`.